echopype/convert/set_groups_ek60.py

```diff
@@ -308,15 +312,27 @@ def set_platform(self, NMEA_only=False) -> xr.Dataset:
                         },
                     },
                     coords={
-                        "ping_time": (
-                            ["ping_time"],
+                        "time2": (
+                            ["time2"],
+                            self.parser_obj.ping_time[ch],
+                            {
+                                "axis": "T",
+                                "long_name": "Timestamps for position datagrams",
+                                "standard_name": "time",
+                                "comment": "Time coordinate corresponding to platform sensors.",
+                            },
+                        ),
+                        "time3": (
+                            ["time3"],
                             self.parser_obj.ping_time[ch],
                             {
                                 "axis": "T",
                                 "long_name": "Timestamps for position datagrams",
```

Comment:
`long_name` may need to be changed, otherwise it's identical (indistinguishable) to the one in `time2`. Maybe "Timestamps for environmental variables"? Though would change the nature of the `long_name` from the data origin (datagram type) to, umm, its scope.